302-signing-workflows.md

```diff
@@ -0,0 +1,40 @@
+---
+title: CNAB Security: Signing workflows
+weight: 302
+---
+
+# Cloud Native Application Bundles Security (CNAB-Sec): Signing workflows 1.0 WD
+
+* [Signing workflow for the minimum viable product (MVP)](#signing-workflow-for-the-minimum-viable-product-mvp)
+
+This document is a _normative_ part of [CNAB Security](300-CNAB-security.md).
+
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119). The use of these keywords in this document are statements about how a CNAB implementation may fulfill the CNAB Security specification _only_.
+
+## Signing workflow for the minimum viable product (MVP)
+
+This subsection documents how a signing workflow MAY typically work for an [MVP metadata repository](301-metadata-repositories.md).
```

Comment:
Using MAY at this high-level context makes it sound like the entire document is non-normative. If that is the case, we should say so. Otherwise, we should explain in what way it is normative.